plugins/source/docs.go

```diff
@@ -21,13 +22,24 @@ var templatesFS embed.FS
 var reMatchNewlines = regexp.MustCompile(`\n{3,}`)
 var reMatchHeaders = regexp.MustCompile(`(#{1,6}.+)\n+`)
 
+func sortTables(tables schema.Tables) {
+	sort.SliceStable(tables, func(i, j int) bool {
+		return tables[i].Name < tables[j].Name
+	})
+
+	for _, table := range tables {
+		sortTables(table.Relations)
+	}
+}
+
 // GeneratePluginDocs creates table documentation for the source plugin based on its list of tables
 func (p *Plugin) GeneratePluginDocs(dir, format string) error {
 	if err := os.MkdirAll(dir, os.ModePerm); err != nil {
 		return err
 	}
 
 	destination.SetDestinationManagedCqColumns(p.Tables())
+	sortTables(p.Tables())
```

Comment:
I think it's a bit unexpected that the `GeneratePluginDocs` function changes the ordering of the tables for the plugin. Maybe `p.Tables()` should rather always return a sorted list?

Reply:
Either that, or we should make a copy of the tables slice, sort it, and pass it in to the render functions

Reply:
> Maybe p.Tables() should rather always return a sorted list?

I think that would be a bigger change as it could impact our scheduling (if we rely on that order to decide which tables go first).

A copy works 👍